kv/union_store.go

```diff
@@ -127,7 +127,7 @@ func (us *UnionStore) CheckLazyConditionPairs() error {
 	for it := us.lazyConditionPairs.NewIterator(nil); it.Valid(); it.Next() {
 		if len(it.Value()) == 0 {
 			if _, exist := values[it.Key()]; exist {
-				return errors.Trace(ErrKeyExists)
```

Comment:
Some of our codes depend on the type of error. For example:  https://github.com/pingcap/tidb/blob/master/table/tables/tables.go#L348. User errors.New intead of predefined error is not a right way.
I think it is better to define a new error in terror package. When raise an error, you can set the code/class/message.